catalog.html

```diff
@@ -152,8 +153,8 @@ <h3 class="hotel-card-title">Amara Resort & Spa</h3>
             </a>
             <b class="hotel-card-price">от 4000 ₽ </b>
             <a href="#" class="hotel-card-button">Подробнее</a>
-            <a href="#" class="hotel-card-button button">В избранное</a>
-            <img class="hotel-rating-star" src="" alt="">
+            <a href="#" class="hotel-card-button-blue ">В избранное</a>
+            <span class="visually-hidden">4 звезды</span>
```

Comment:
Не учла только что нам нужно еще и стилизовать что-то будет для оформления звезд:
```suggestion
            <p class="stars stars-4">
              <span class="visually-hidden">4 звезды</span>
            </p>
```

Reply:
Думала над этим, добавила.